.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+
+node_js: "6"
+
+addons:
+  apt:
+    packages:
+      - xvfb
+      - fakeroot
+      - dpkg
+      - libdbus-1-dev
+      - libglib2.0-dev
+      - dbus
+      
+before_install:
+  - cd debian/
+  
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  
```

Comment:
If there's no GUI test run on Travis. Then it's better to remove it. I'll go ahead and do that.